src/contracts/BuildersManager.sol

```diff
@@ -152,18 +146,37 @@ contract BuildersManager is EIP712Upgradeable, Ownable2StepUpgradeable, IBuilder
 
     uint256 _yield = TOKEN.yieldAccrued();
     TOKEN.claimYield(_yield);
-    uint256 _yieldPerProject = ((_yield * _PRECISION) / _l) / _PRECISION;
+    uint256 _yieldPerProject = ((_yield * WAD) / _l) / WAD;
 
     for (uint256 _i; _i < _l; _i++) {
       TOKEN.transfer(_currentProjects[_i], _yieldPerProject);
     }
     emit YieldDistributed(_yieldPerProject, _currentProjects);
   }
 
+  /// @inheritdoc IBuildersManager
+  function registerSchema(bytes32 _schemaUid, address _validator) external onlyOwner {
```

Comment:
long term we may want to enable optimism citizens to vote on adding a new schema, or letting optimism foundation attestors add new schemas - prob out of scope